src/test/OpenEXRCoreTest/CMakeLists.txt

```diff
@@ -40,10 +40,11 @@ if(WIN32 AND (BUILD_SHARED_LIBS OR OPENEXR_BUILD_BOTH_STATIC_SHARED))
   target_compile_definitions(CorePerfTest PRIVATE OPENEXR_DLL)
 endif()
 
-#add_test(NAME OpenEXR.Core COMMAND $<TARGET_FILE:OpenEXRCoreTest>)
```

Comment:
Looking at this closer, the original line was dead commented-out code and should be removed entirely (thee #add_test line).